packages/terra-layout/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Layout
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-layout.svg)](https://www.npmjs.org/package/terra-layout)
+[![Build Status](https://travis-ci.org/cerner/terra.svg?branch=master)](https://travis-ci.org/cerner/terra-framework)
+
+This component renders secondary navigation items with react router.
```

Comment:
Is this description correct. Renders secondary nav items? I'd think something `This component renders an application layout framework with react router`

Reply:
Or I just saw this below: `The Layout component provides a responsive starting point for the position of high-level application components.`